doc/changes/changelog.md

```diff
@@ -1,5 +1,6 @@
 # Changes
 
+* [0.1.13](changes_0.1.13.md)
 * [0.1.12](changes_0.1.12.md)
 * [0.1.11](changes_0.1.11.md)
 * [0.1.10](changes_0.1.10.md)
```

Comment:
.gitattributes?

Reply:
No, pk does not support npm yet ;)

Reply:
See https://github.com/exasol/project-keeper/issues/373